Content.Server/Zombies/ZombieSystem.cs

```diff
@@ -156,24 +159,102 @@ private void DoGroan(EntityUid uid, ActiveZombieComponent component)
             component.LastDamageGroanCooldown = component.GroanCooldown;
         }
 
+        private void OnZombieInit(EntityUid ent, ZombieComponent component, ComponentStartup _)
+        {
+            // Brute
+            component.HealingDamageSpecifier.DamageDict["Brute"] = -10;
+            component.HealingDamageSpecifier.DamageDict["Pierce"] = -10;
+            component.HealingDamageSpecifier.DamageDict["Blunt"] = -10;
+
+            // Burn
+            component.HealingDamageSpecifier.DamageDict["Heat"] = -10;
+            component.HealingDamageSpecifier.DamageDict["Shock"] = -10;
+            component.HealingDamageSpecifier.DamageDict["Cold"] = -10;
+
+            // Zombies do not receive any Toxic or Airloss damage
+        }
```

Comment:
This shouldn't be done on startup and zombies shouldn't be healing 60 damage every 30 seconds.

Reply:
They heal up to 60 damage. Usually they will heal 10-20